custom/icds_reports/migrations/0035_auto_20180317_0645.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-03-17 06:45
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0035_add_pnc_aggregate_models'),
+    ]
+
+    operations = [
+        migrator.get_migration('update_tables17.sql'),
```

Comment:
Hi @emord I think you should add in this place also ```migrator.get_migration('create_functions.sql')``` without this, we will have the old ```insert_into_child_health_monthly``` definition on the production.

Reply:
@lwyszomi we run `create_functions` every time we aggregate https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tasks.py#L83-L96

Likely something that should be changed, but for now it's not an issue

Reply:
@emord didn't notice. I thought that the task runs only aggregation functions.